### PR TITLE
replace ecology with react-live for homepage demos

### PR DESCRIPTION
**Pull request by redfieldstefan**
Replaces `ecology` with `react-live` for home page demos to avoid webpack build-static errors being caused by `ecology` library (failing in Travis). 

[https://gyazo.com/749945b09f821083f753844db544430b](https://gyazo.com/749945b09f821083f753844db544430b)

### CODE REVIEW

File: src/radium-stylesheet.js [hunk 1]
```diff
@@ -245,6 +245,39 @@ export default {
...
+  ".react-live-preview": {
+    position: "relative",
+    "text-align": "center",
+    order: "0",
+    padding: "40px 20px",
+    background: "#eee",
+    border: "1px solid #cecece"
+  },
+  ".prism-code.LiveEditor": {
+    background: theme.codeMirror.bg,
```
Comment (kylecesmat) [inline src/radium-stylesheet.js]: we should rename this `codeMirror` object and remove unused codemirror styles.

**Comment by redfieldstefan** [comment]
I just observed a styling side effect that the `react-live` dependency brings in... the color is removed from the code samples in api docs

Current:
screen shot 2017-05-05 at 4 36 03 pm

With `react-live`
screen shot 2017-05-05 at 4 36 26 pm

**Comment by redfieldstefan** [comment]
@kylecesmat Done and done, let me know if I can adjust anything else.

**Comment by paulathevalley** [comment]
I _think_ the `/static/prism-github-dark.css` is the stylesheet you want `react-live` to use, perhaps using `mountStylesheet`...?

**Comment by kylecesmat** [comment]
yeah, we could use that github stylesheet or a stylesheet of your choice. Here are some examples: https://github.com/PrismJS/prism-themes

The styles are applied from the `.prism-code` root. So either loading it via a stylesheet as @paulathevalley suggests or copying the style into our own stylesheet works too, whichever you prefer!

File: src/radium-theme.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ module.exports = {
   // ---------------------
   // CodeMirror Theme
```
Comment (kylecesmat) [inline src/radium-theme.js]: go ahead and rename this too

**Comment by kylecesmat** [comment]
codemirror is also being brought in here too https://github.com/FormidableLabs/radium-docs/blob/master/templates/index.hbs#L20

**Comment by redfieldstefan** [comment]
@kylecesmat @paulathevalley Importing/using just `hightlightAll` from `prism-core` seems to fix the missing syntax highlighting in docs that come with updated `Prism` dependency.

https://github.com/FormidableLabs/radium-docs/pull/45/files#diff-5f29916fabdf60f7af8e5e915a455853R8

screen shot 2017-05-08 at 9 52 57 am

**Comment by philpl** [comment]
I haven't done a lot yet around prism stylesheets and react live. If you need anything to change, I'm here to help :)

**Comment by redfieldstefan** [comment]
@philpl Thanks! At this juncture we're all set I think, this latter issue was just some other, non-live, code examples not playing well with the newer version of `Prism`. The `react-live` examples are looking great!

**Pull request merged**